src/views/templates/list/VirtualMachineTemplatesList.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import {
+  TEMPLATE_TYPE_BASE,
+  TEMPLATE_TYPE_LABEL,
+  TEMPLATE_TYPE_VM,
+} from 'src/views/catalog/templatescatalog/utils/constants';
```

Comment:
we should not use information from another extension.
we may want to remove the catalog, change it, remove/change this consts in the catalog some day.

If you need this consts for this extension, add them in this extension

Reply:
Thank you for pointing this, you are right, not a good practice - importing something from some other extension. And I would better move those constants to some more generic place, for the reasons I've already mentioned in my another comment.